code/game/machinery/autolathe.dm

```diff
@@ -341,7 +341,7 @@
 	. = ..()
 	var/mat_capacity = 0
 	for(var/datum/stock_part/matter_bin/new_matter_bin in component_parts)
-		mat_capacity += new_matter_bin.tier * 75000
+		mat_capacity += new_matter_bin.tier * (37.5*SHEET_MATERIAL_AMOUNT)
```

Comment:
This is scaled correctly because the old values of 
SHEET_MATERIAL_AMOUNT was 2000. Apply the same scaling for all values everywhere else